scripts/null_route_helper

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python3
+
+"""
+Utility for blocking and unblocking traffic from given source ip_prefix on ACL tables.
+
+The block operation will insert a DENY rule at the top of the table. The unblock operation
+will remove an existing DENY rule that has been created by the block operation (i.e. it does
+NOT insert an ALLOW rule, only removes DENY rules).
+
+Since SONiC supports multi ACL rules share the same priority, all ACL rules created by null_route_helper will
+use the highest priority(9999).
+
+Example:
+
+Block traffic from 10.2.3.4/32:
+./null_route_helper acl_table_name block 10.2.3.4/32
+
+Unblock all traffic from 10.2.3.4/32:
+./null_route_helper acl_table_name unblock 10.2.3.4/32
```

Comment:
>unblock [](http://example.com/codeflow?start=35&length=7)

There is no action to list all the exising rules. So if a user want to `unblock` all, it's inconvenient

Reply:
It's a good suggestion. Thanks.
I was thinking which one is better, a ```list``` command or a ```unblock all``` command? Any suggestions?

Reply:
`list` command will be useful in many cases, including
1. unblock all
2. verify existing rules

Reply:
I added a new interface `list`. Thanks

Reply:
Add the `list` command in the usage comment?